docs/sources/installation/security.md

```diff
@@ -12,29 +12,27 @@ weight = 2
 
 # Security
 
-## Data source proxy and protecting internal services
+If you run non-Grafana web services on your Grafana server or within its local network, then they might be vulnerable to exploitation through the Grafana data source proxy.
 
-If you have non-Grafana web services running on your Grafana server or within its local network, these may be vulnerable to exploitation via the Grafana data source proxy.
+To prevent this type of exploitation from happening, we recommend that you apply one or more of the precautions listed below.
 
-To prevent this type of exploitation from happening we explain a couple of different solutions  below.
+## Limit IP addresses/hostnames for data source URL
 
-### Configure Grafana to only allow certain IP addresses/hostnames to be used as data source url
+You can configure Grafana to only allow certain IP addresses or hostnames to be used as data source URL and by that proxied through the Grafana data source proxy. Refer to [data_source_proxy_whitelist]({{< relref "configuration/#data-source-proxy-whitelist" >}}) for usage instructions.
 
-You can configure Grafana to only allow certain IP addresses/hostnames to be used as data source url and by that proxied through the Grafana data source proxy. See [data_source_proxy_whitelist]({{< relref "configuration/#data-source-proxy-whitelist" >}}) for usage instructions.
+## Firewall rules
 
-### Firewall rules
+Configure a firewall, for example using iptables, to restrict Grafana from making network requests to certain internal web services.
 
-You should be able to configure a firewall, for example using iptables, to restrict Grafana from making network requests to certain internal web services.
+## Proxy server
 
-### Proxy server
-
-You should be able to require all network requests being made by Grafana to go through a proxy server.
+Require all network requests being made by Grafana to go through a proxy server.
 
 ## Viewer query permissions
 
-Important to understand that users with Viewer role can still issue any possible query to all data sources available in the **organization**. Not just the queries that are defined on the dashboards the user with Viewer role has permissions to view.
+Users with the Viewer role can still issue any possible query to all data sources available in the **organization**, not just the queries that are defined on the dashboards the user with Viewer role has permissions to view.
```

Comment:
End of this sentence (after comma) feels weird to me, but maybe me not used to reading/writing in this way?

Reply:
No, it is a terrible sentence. I am just not sure what it is trying to say, so not sure how to rewrite it.

Reply:
It is saying that a person with the viewer role can issue any query they want (if they have some IT skills). An admin might think that viewers can only issue queries that are defined in the dashboards but that would be a false sense of security.

For example, if there is a Grafana instance with one data source, one dashboard and one panel which has one query defined then you might think that is all a viewer can see. But a viewer has access to send queries to the data source so with a command-line tool like curl (there are lots of tools for this), the viewer can make their own query and send it to the data source and potentially access sensitive data.

Reply:
Updated